core/server/web/shared/middlewares/emit-events.js

```diff
@@ -0,0 +1,12 @@
+const common = require('../../../lib/common');
+
+module.exports = function emitEvents(req, res, next) {
+    res.on('finish', function triggerEvents() {
+        if (res.get('X-Cache-Invalidate') === '/*') {
```

Comment:
Maybe instead of using a magic-string with '/*' would be nice to use [INVALIDATE_ALL](https://github.com/TryGhost/Ghost/blob/959912eca316e12715808b1cbaa7bc55ebfb81cb/core/server/api/shared/headers.js#L2)? :man_shrugging: